.docker/readeef/scripts/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#/bin/sh
+
+set -x 
+set -e
+
+case "$1" in
+
+  'server')
+  	exec readeef server $@ $ARGS
+	;;
+
+  'client')
+  	exec readeef-client $@ $ARGS
+	;;
```

Comment:
you can omit that. its a highly experimental webview desktop client. i don't think it has any place in a docker container

Reply:
removed